app/templates/components/popular-tags.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="col-md-3">
+  <div class="sidebar">
```

Comment:
I'm with @jonnii here. I don't think this component is actually necessary at all. There's really no need to make a component for something that's only used one time.